src/components/forms/AddTransactionsForm.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+
+import PriceDescriptionList from "../formLists/PriceDescriptionList";
+import {
+  TextField,
+  Stack,
+  InputAdornment,
+  Button,
+  AvatarGroup,
+  Avatar,
+} from "@mui/material";
+import AttachMoneyIcon from "@mui/icons-material/AttachMoney";
+import Calculate from "../calculate/Calculate";
+
+const title = { fontSize: "13px", fontWeight: "bold" };
+
+let getTransactionData = () => {
+  const data = JSON.parse(localStorage.getItem("transactions"));
+  if (data !== null) {
+    return data;
+  }
+  return [];
+};
+
+const AddTransactionsForm = ({
+  activityDetails,
+  setIsTransactionFormShown,
+}) => {
+  const [description, setDescription] = useState("");
+  const [price, setPrice] = useState("");
+  const [payer, setPayer] = useState(activityDetails.items[0]);
+  const [details, setDetails] = useState(getTransactionData);
+
+  let changes = {
+    description: description,
+    price: price,
+    payer: payer,
+  };
+
+  const addItems = () => {
+    let storage = [];
```

Comment:
```suggestion
    const storage = [];
```

arrays and objects always const